deploy/ansible/roles/k8s_resources/tasks/dashboard.yaml

```diff
@@ -36,8 +36,8 @@
 - name: Create kubernetes-dashboard ingress configuration
   template:
     src: "dashboard.ing.yml.j2"
-    dest: "/tmp/dashboard.ing.yml"
+    dest: "{{ tmp_dir }}/dashboard.ing.yml"
```

Comment:
Maybe we should add cluster name here too?

Reply:
done